metadata.rb

```diff
@@ -3,15 +3,16 @@
 maintainer_email 'cookbooks@chef.io'
 license 'Apache-2.0'
 description 'Manage Splunk Enterprise or Splunk Universal Forwarder'
-version '1.7.3'
+long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
```

Comment:
long_description has been deprecated and can be removed

Reply:
feature request?

Reply:
?

Reply:
removed long_description